packages/tools/devtools/devtools-view/src/components/ContainerHistoryLog.tsx

```diff
@@ -31,6 +31,7 @@ import { LabelCellLayout } from "./utility-components";
  * Returns the text color based on the current color theme of the devtools.
  */
 function setThemeStyle(themeName: string, state: string): string {
```

Comment:
I think we can safely update the `themeName` parameter to be of type `ThemeOption`